packages/@expo/cli/src/start/server/metro/runServer-fork.ts

```diff
@@ -68,6 +71,18 @@ export const runServer = async (
     httpServer = http.createServer(serverApp);
   }
 
+  let connections: Socket[] = [];
```

Comment:
Could use a Set if you want to make this const and not need to call filter (can call remove() instead)